main.go

```
@@ -136,6 +136,9 @@ func (w *Walker) walk(inputPath, outputPath string, depth, maxDepth int, visited
 			if hashGenerated != hash || emptyManifest {
 				log.Printf("No match detected. Render: %s\n", crd.ObjectMeta.Name)
 				if err := w.Render(crd, path); err != nil {
+					if strings.Contains(err.Error(), "not supported") {
```

Instead of returning below a `kustomize not supported`, maybe throw a specific app error so the above code can be changed to `errors.Is` for more precise error handling.

---

I chose this method in case we wanted to add another "not supported" case, it would automatically work.

---

Changed to use the `errors.Is`, and I used `errors.ErrUnsupported`.